clang/lib/AST/CommentParser.cpp

```diff
@@ -89,6 +108,114 @@ class TextTokenRetokenizer {
     }
   }
 
+  bool continueInt(SmallString<32> &NextToken) {
+    return NextToken.ends_with(StringRef("char")) ||
```

Comment:
Don't really understand what this is for...

Reply:
This is used during integral lexing to decide whether we should continue lexing the presumed integral type or not. The name was confusing, so I've renamed it from `continueInt` to `shouldContinueLexingIntegralType`